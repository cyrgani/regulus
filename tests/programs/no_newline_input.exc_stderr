IoError: missing newline after input() call
at programs/no_newline_input.re:1:37
at programs/no_newline_input.re:1:31

