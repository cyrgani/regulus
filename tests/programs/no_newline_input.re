__builtin_print_catch(input()),