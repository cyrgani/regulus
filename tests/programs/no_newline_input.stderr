IoError: missing newline after input() call
at programs/no_newline_input.re:1:28
at programs/no_newline_input.re:1:22
